src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OverrideSchedulingListingFields.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OverrideSchedulingListingFields {
+
+    @JsonProperty("overrideDuration")
```

Comment:
No need for @JsonProperty if it matches the field name

Reply:
didnt realise this - thanks!